lib/components/artist/shows.js

```diff
@@ -0,0 +1,12 @@
+/* @flow */
+'use strict';
+
+import React from 'react-native';
+const { Text } = React;
+
+
+export default class Shows extends React.Component {
+	render() {
+		return <Text> Current & Upcoming Shows </Text>
+	}
```

Comment:
Finally, I tend to wrap my [JSX](https://facebook.github.io/jsx/) in parentheses, which is needed for multiple lines of JSX.